cpp/translator.cc

```diff
@@ -243,16 +245,90 @@ bool IsEmptyEntryWithoutTranslator(const std::string &entry) {
 }
 
 
+void GetTranslatorLanguages(const IniFile &ini_file, const std::string &translator,
+                            std::vector<std::string> * const translator_languages)
+{
+    // If user is an administrator all languages are open for editing, otherwise only the specified ones
+    const std::string ini_administrators(ini_file.getString(USER_SECTION, "administrators"));
+    std::vector<std::string> administrators;
+    StringUtil::Split(ini_administrators, ",", &administrators);
```

Comment:
When splitting on a single character, please use the version that uses a char for the delimiter.